cmake/CMakeLists_posix.txt

```diff
@@ -160,6 +177,34 @@ endforeach()
 
 install(TARGETS px4 DESTINATION bin)
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/ROMFS DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
-install(DIRECTORY ${PROJECT_SOURCE_DIR}/posix-configs DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/posix-configs/SITL/init DESTINATION ${CMAKE_BINARY_DIR}/rootfs/etc/init)
```

Comment:
@julianoes  I think it would be worthwhile to rebase on https://github.com/PX4/Firmware/pull/5255 now.